images/win/scripts/Installers/Install-MysqlCli.ps1

```diff
@@ -11,9 +11,16 @@ $ArgumentList = ("/install", "/quiet", "/norestart")
 Install-Binary -Url $InstallerURI -Name $InstallerName -ArgumentList $ArgumentList
 
 ## Downloading mysql
-$MysqlMajorVersion = (Get-ToolsetContent).mysql.major_version
-$MysqlFullVersion = (Get-ToolsetContent).mysql.full_version
-$MysqlVersionUrl = "https://dev.mysql.com/get/Downloads/MySQL-${MysqlMajorVersion}/mysql-${MysqlFullVersion}-winx64.zip"
+[version]$MysqlVersion = (Get-ToolsetContent).mysql.version
+$MysqlVersionMajorMinor = $MysqlVersion.ToString(2)
+
+if ($MysqlVersion.Build -lt 0) {
+    $MysqlVersion = ((Invoke-WebRequest -Uri https://dev.mysql.com/downloads/mysql/${MysqlVersionMajorMinor}.html -UseBasicParsing).Content |
```

Comment:
What do you think to replace Invoke-WebRequest -> Invoke-RestMethod and remove ().Content?